### PR TITLE
Block Builder mappings

### PR DESCRIPTION
**Pull request by Prospector**
Ok, ignore the massive amount of files changed. That's a cascading effect from a single naming of a parameter. Really, the entire PR is just a few methods and fields in Block.Builder. plz merg quik

### CODE REVIEW

File: mappings/net/minecraft/block/Block.mapping [hunk 1]
```diff
@@ -9,30 +9,58 @@ CLASS bfg net/minecraft/block/Block
...
 		METHOD a copy (Lbfg;)Lbfg$c;
+			ARG 0 block
+			ARG 1 builder
 		METHOD a soundGroup (Lbkh;)Lbfg$c;
+			ARG 1 soundGroup
 		METHOD a create (Lccw;)Lbfg$c;
 		METHOD a create (Lccw;Laub;)Lbfg$c;
 		METHOD a create (Lccw;Lccx;)Lbfg$c;
+		METHOD b breakInstantly ()Lbfg$c;
+		METHOD b breakValues (F)Lbfg$c;
```
Comment (asiekierka) [inline mappings/net/minecraft/block/Block.mapping:40]: break... Values?

File: mappings/net/minecraft/block/Block.mapping [hunk 2]
```diff
@@ -9,30 +9,58 @@ CLASS bfg net/minecraft/block/Block
...
+			ARG 1 value
 		METHOD b copyDropTable (Lbfg;)Lbfg$c;
+			ARG 1 block
+		METHOD c randomlyTicks ()Lbfg$c;
```
Comment (asiekierka) [inline mappings/net/minecraft/block/Block.mapping]: ticksRandomly

File: mappings/net/minecraft/block/Block.mapping [hunk 3]
```diff
@@ -9,30 +9,58 @@ CLASS bfg net/minecraft/block/Block
...
 		FIELD a material Lccw;
 		FIELD b mapColor Lccx;
+		FIELD c collidable Z
 		FIELD d soundGroup Lbkh;
-		FIELD e lightOpacityLevel I
+		FIELD e luminance I
+		FIELD f resistance F
+		FIELD g hardness F
+		FIELD h randomTicks Z
+		FIELD i slipperiness F
```
Comment (modmuss50) [inline mappings/net/minecraft/block/Block.mapping:18]: friction? Not sure about slipperiness

**Comment by asiekierka** [comment]
Tackled separately.

**Pull request closed without merging**